drivers/mbox/Kconfig.nxp_fsl

```diff
@@ -0,0 +1,8 @@
+# Copyright 2022 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config MBOX_NXP_FSL_MU
+	bool "NXP Freescale Message Unit (MR) driver"
+	default n
```

Comment:
This should be:
```
default y
depends on DT_HAS_...
```
Please, check how this is done for the other MBOX drivers.

Reply:
Done